README.md

```diff
@@ -85,6 +96,31 @@ func main() {
 }
 ```
 
+## Getting Help
+
+Please use these community resources for getting help. We use the GitHub issues
+for tracking bugs and feature requests.
+
+* Ask a question on [StackOverflow](http://stackoverflow.com/) and tag it with the [`aws-sdk-go`](http://stackoverflow.com/questions/tagged/aws-sdk-go) tag.
+* Come join the AWS SDK for Go community chat on [gitter](https://gitter.im/aws/aws-sdk-go).
```

Comment:
Nit this is the v1 channel. there is an `aws-sdk-go-v2` channel as we.. But may make sense to use just one for more collaboration.

Reply:
Fixed! If we decide to merge the two channels later on we can update this link accordingly.